packages/frontend-2/components/viewer/Controls.vue

```diff
@@ -391,6 +391,8 @@ watch(isSmallerOrEqualSm, (newVal) => {
 })
 
 onKeyStroke('Escape', () => {
-  activeControl.value = 'none'
+  if (activeControl.value !== 'measurements') {
```

Comment:
Not ideal that the logic for what should happen on Esc is split apart in multiple files (this and postSetup.ts), but it can be left as is for now

Reply:
Now in 1 file - Controls.vue